frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -40,7 +40,7 @@ const RIGHT_SIDEBAR_CLOSE_BUTTON_TEST_ID = "right-sidebar-close-button";
 const GENE_INFO_BUTTON_CELL_INFO_TEST_ID = "gene-info-button-cell-info";
 
 // Export constants
-const CSV_NUM_METADATA_ROWS = 14;
+const CSV_START_FROM_ROW_NUM = 9; // This is the number of metadata rows + 1
```

Comment:
not sure why it was changed to 14 in [this PR](https://github.com/chanzuckerberg/single-cell-data-portal/pull/4483/files), but it broke the tests. So I changed it back to 9 for now. Reached out to Victor about it already